contrib_bots/lib/joker.py

```diff
@@ -0,0 +1,52 @@
+# pip install schedule is necessary
+import schedule
+import time
+import json
+import threading
+import os
+from random import choice
+
+cache = os.path.expanduser('lib/Joker/jokes.json')
+
+def job(client):
+    with open(cache) as data_file:
+        joke = list(choice(json.load(data_file)).values())[0]
```

Comment:
Choice returns a dictionary, and for some reason choice.values() returns the message in brackets [].